server/include/server.h

```diff
@@ -9,6 +9,9 @@
 #ifndef ZAPPY_SERVER_H
     #define ZAPPY_SERVER_H
 
+    #define MAX_COMMAND_SIZE 1024
```

Comment:
How do you know the max command size is 1024 ?
Do that mean i cannot send a message with broadcast that is longer than that ?

Reply:
What about this ?

Reply:
It's an arbitrary limit